packages/hub/tasks/notify-customer-payment.ts

```diff
@@ -65,9 +69,23 @@ export default class NotifyCustomerPayment {
       );
     }
 
+    let merchantName = '';
+
+    try {
+      if (result.merchantSafe?.infoDid) {
+        let merchantInfo = await this.merchantInfo.getMerchantInfo(result.merchantSafe.infoDid);
+
+        if (merchantInfo) {
```

Comment:
Maybe worth changing to `merchantInfo?.name`?

Reply:
Agreed, done in c91e5682a, thanks